MatrixSDK/Utils/MXTools.m

```diff
@@ -70,74 +70,145 @@ + (void)initialize
+            @(MXEventTypeRoomTag) : kMXEventTypeStringRoomTag,
+            @(MXEventTypeRoomTombStone) : kMXEventTypeStringRoomTombStone,
+            
+            @(MXEventTypePresence) : kMXEventTypeStringPresence,
+            @(MXEventTypeTypingNotification) : kMXEventTypeStringTypingNotification,
+            @(MXEventTypeReaction) : kMXEventTypeStringReaction,
+            @(MXEventTypeReceipt) : kMXEventTypeStringReceipt,
+            @(MXEventTypeRead) : kMXEventTypeStringRead,
+            @(MXEventTypeReadMarker) : kMXEventTypeStringReadMarker,
+            @(MXEventTypeSticker) : kMXEventTypeStringSticker,
+            @(MXEventTypeTaggedEvents) : kMXEventTypeStringTaggedEvents,
+            @(MXEventTypeSpaceChild) : kMXEventTypeStringSpaceChild,
+            
+            @(MXEventTypeCallInvite) : kMXEventTypeStringCallInvite,
+            @(MXEventTypeCallCandidates) : kMXEventTypeStringCallCandidates,
+            @(MXEventTypeCallAnswer) : kMXEventTypeStringCallAnswer,
+            @(MXEventTypeCallSelectAnswer) : kMXEventTypeStringCallSelectAnswer,
+            @(MXEventTypeCallHangup) : kMXEventTypeStringCallHangup,
+            @(MXEventTypeCallReject) : kMXEventTypeStringCallReject,
+            @(MXEventTypeCallNegotiate) : kMXEventTypeStringCallNegotiate,
+            @(MXEventTypeCallReplaces) : kMXEventTypeStringCallReplaces,
+            @(MXEventTypeCallRejectReplacement) : kMXEventTypeStringCallRejectReplacement,
+            @(MXEventTypeCallAssertedIdentity) : kMXEventTypeStringCallAssertedIdentity,
+            @(MXEventTypeCallAssertedIdentityUnstable) : kMXEventTypeStringCallAssertedIdentityUnstable,
+            
+            @(MXEventTypeKeyVerificationRequest) : kMXEventTypeStringKeyVerificationRequest,
+            @(MXEventTypeKeyVerificationReady) : kMXEventTypeStringKeyVerificationReady,
+            @(MXEventTypeKeyVerificationStart) : kMXEventTypeStringKeyVerificationStart,
+            @(MXEventTypeKeyVerificationAccept) : kMXEventTypeStringKeyVerificationAccept,
+            @(MXEventTypeKeyVerificationKey) : kMXEventTypeStringKeyVerificationKey,
+            @(MXEventTypeKeyVerificationMac) : kMXEventTypeStringKeyVerificationMac,
+            @(MXEventTypeKeyVerificationCancel) : kMXEventTypeStringKeyVerificationCancel,
+            @(MXEventTypeKeyVerificationDone) : kMXEventTypeStringKeyVerificationDone,
+            
+            @(MXEventTypeSecretRequest) : kMXEventTypeStringSecretRequest,
+            @(MXEventTypeSecretSend) : kMXEventTypeStringSecretSend,
+            @(MXEventTypeSecretStorageDefaultKey) : kMXEventTypeStringSecretStorageDefaultKey,
+            
+            @(MXEventTypePollStart) : kMXEventTypeStringPollStartMSC3381,
+            @(MXEventTypePollResponse) : kMXEventTypeStringPollResponseMSC3381,
+            @(MXEventTypePollEnd) : kMXEventTypeStringPollEndMSC3381,
+        };
+
+        eventTypeMapStringToEnum = @{
```

Comment:
It's a shame one of these dictionaries can't be generated from the other one, but I can see why 😕